STORAGE_CLASS_README.md

```diff
@@ -0,0 +1,132 @@
+# Development of storage class template 
+The storage class template is created in `storage-class-template.yaml` file as per the directory structure mentioned [here]README.md#filedirectory-reference).
+This file defines the template that contain the configuration of storage classes in YAML format. It is Kubernetes `StorageClass` resource that contains the set of storage class parameters and allowed variable substitutions.  The  example of `storage-class-template.yaml` is as below 
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "{{ name }}"
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "{{ is-default-class }}"
+provisioner: block.csi.ibm.com
+parameters:
+  SpaceEfficiency: "{{ space-efficiency }}"   # Optional.
+  pool: "{{ pool }}" 
+
+  csi.storage.k8s.io/provisioner-secret-name: "{{ secret-name }}"
+  csi.storage.k8s.io/provisioner-secret-namespace: "{{ secret-namespace }}"
+  csi.storage.k8s.io/controller-publish-secret-name: "{{ secret-name }}"
+  csi.storage.k8s.io/controller-publish-secret-namespace: "{{ secret-namespace }}"
+  csi.storage.k8s.io/controller-expand-secret-name: "{{ secret-name }}"
+  csi.storage.k8s.io/controller-expand-secret-namespace: "{{ secret-namespace }}"
+
+  csi.storage.k8s.io/fstype: "{{ fstype }}"   # Optional. Values ext4\xfs. The default is ext4.
+  volume_name_prefix: "{{ prefix }}"      # Optional.
+allowVolumeExpansion: {{ VolumeExpansion }} #false
+```
+
+We DO NOT use `MustachTemplate` engine to translate the storage class template to actual storage class configuration. Rather, we just replace  ` {{ varaibale-name }}` with its value given by the user. The multiple storage classes can be created from the same storage-class-template. The user is allowed to pass the array of storage class parameters to create multiple storage classes. 
```

Comment:
Also mention for storage-class template triple mustach `{{{ ... }}}` is invalid notation for parameter replacement.